controllers/AuthController.js

```diff
@@ -92,32 +86,34 @@ class AuthController {
   static async login(req, res) {
     const { email, password } = req.body;
 
-    const user = await UserModelQuery.getUserByEmail(email);
-    if (!user) {
-      const payload = {
-        message: 'email does not exist'
-      };
-      return statusResponse.conflict(res, payload);
-    }
-    if (!bcrypt.compareSync(password, user.dataValues.password)) {
+    try {
+      const user = await UserModelQuery.getUserByEmail(email);
+      if (!user) {
+        const payload = {
+          message: 'email does not exist'
+        };
+        return StatusResponse.conflict(res, payload);
```

Comment:
Why is the response of type conflict here? If the user does not exist, it shouldn't be a conflict.